tests/components/AppliedFilters.stories.tsx

```diff
@@ -5,15 +5,15 @@ import { AnswersHeadlessContext, SearchTypeEnum } from '@yext/answers-headless-r
 import { generateMockedHeadless } from '../__fixtures__/answers-headless';
 import { AppliedFiltersDisplay } from '../../src/components/AppliedFiltersDisplay';
 import { DisplayableFilters, DisplayableHierarchicalFacets } from '../__fixtures__/data/filters';
-import { builtInCssClasses } from '../../src/components/AppliedFilters';
+import { AppliedFiltersProps, builtInCssClasses } from '../../src/components/AppliedFilters';
```

Comment:
why can't we make `hierarchicalFacetsDelimiter` configurable from the pr description? Also, I think it may be better to avoid confusion if we disable control configuration for `geolocationOptions`, `sectioned`, and `searchFields` since the response is mocked a specific way at the moment.

Reply:
`hierarchicalFacetsDelimiter` in `AppliedFiltersDisplay` is only used when removing an applied filter and updating the hierarchical filters accordingly. But, since we are passing in hard-coded filters directly into `AppliedFiltersDisplay`, the re-computed applied filters are never used or displayed so changing the delimiter has no effect. I'll update to disable all controls that don't have an effect